console_windows.go

```diff
@@ -154,6 +154,10 @@ func (m *master) Fd() uintptr {
 	return m.in
 }
 
+func (m *master) Name() string {
+	return ""
+}
```

Comment:
On windows, the console can only be made from `os.Std{in,out,err}` and not an arbitrary file descriptor like on linux, so the returned value here doesn't matter.

This is due to the fact that a process can only be associated with one Console on windows and usually that's already the case when it's created (I guess that's the reason from the lack of `AllocConsole()/GetStdHandle()` use in the code - it was ported over from the docker codebase -)

I guess you could just return `"console"` here.